src/main/java/ch/njol/skript/conditions/CondRespawnLocation.java

```diff
@@ -0,0 +1,67 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+
+package ch.njol.skript.conditions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.lang.Condition;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import org.bukkit.event.Event;
+import org.bukkit.event.player.PlayerRespawnEvent;
+import org.eclipse.jdt.annotation.Nullable;
+
+
+public class CondRespawnLocation extends Condition {
+
+	static {
+		if (Skript.classExists("org.bukkit.block.data.type.RespawnAnchor"))
+			Skript.registerCondition(CondRespawnLocation.class, "[the] respawn location (was|is)[(1¦(n'| no)t)] [a] (:bed [spawn]|respawn anchor)");
```

Comment:
```suggestion
			Skript.registerCondition(CondRespawnLocation.class, "[the] respawn location (was|is)[1:(n'| no)t] [a] (:bed|respawn anchor)");
```

Some pattern improvements / suggestions 

Just want to also note here that parse tags/marks work anywhere in a pattern, not just choice parts :)